src/PrimaryCensoredDist.jl

```diff
@@ -64,18 +64,23 @@ end
 Base.eltype(::Type{<:PrimaryCensoredDist{D}}) where {D} = promote_type(eltype(D), eltype(D))
 
 function Distributions.cdf(d::PrimaryCensoredDist, x::Real)
-    if x <= 0
-        return 0
+    solver = QuadGKJL()
+    if x <= minimum(d.censoring)
+        return 0.0
     end
 
     function f(u, x)
-        return exp(logcdf(d.uncensored, x) + logpdf(d.censoring, x - u))
+        return exp(logcdf(d.uncensored, x - u) + logpdf(d.censoring, u))
+    end
+
+    domain = (min(minimum(d.censoring), x), min(maximum(d.censoring), x))
```

Comment:
why have you flipped the domain relative to the stan code and what was here before? I think unless there is a pressing reason that should be its own issue and we should change the R package as well (if we don't that means we would now have 3 parameterizations for this)

Reply:
But I've just committed a change to have the same way around